File: backend/Models/Destination.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Flyt.Models
{
    public class Destination
    {
        public int Id { get; set; }
        public int? RouteId { get; set; }
        public Route Route { get; set; }
        public int? StoppointId { get; set; }
        public Stoppoint Stoppoint { get; set; }
        public int Sequence { get; set; }
        public DateTime ETA { get; set; }
        public DateTime ArrivalTime { get; set; }
        public DateTime LeaveTime { get; set;}
        public int Kg { get; set; }
    }
}

File: frontend/src/types/Destination.ts
import { Route } from '@/types/Route';
import { StopPoint } from '@/types/StopPoint';

export interface Destination {
  DestinationId: string;
  Route: Route;
  StopPoint: StopPoint;
  Sekvens: string;
  ETA: Date;
  ArrivalTime: Date;
  LeaveTime: Date;
  Kg: number;
}
